eslint.config.js

```diff
@@ -10,11 +10,9 @@ import jsxA11y from "eslint-plugin-jsx-a11y";
 import react from "eslint-plugin-react";
 import reactHooks from "eslint-plugin-react-hooks";
 import globals from "globals";
-import tseslint from "typescript-eslint";
 
 export default [
   eslint.configs.recommended,
-  ...tseslint.configs.recommended,
```

Comment:
we don't really need typescript-eslint because deno lint already covers a lot of the rules the recommended config provides.